client/devpi/main.py

```diff
@@ -659,6 +659,14 @@ def user(parser):
         help="key=value configuration item.  Possible keys: "
              "email, password.")
 
+
+@subcommand("devpi.user:passwd")
+def passwd(parser):
+    """ change user password. """
+    parser.add_argument(
+        "username", type=str, action="store", nargs="?", help="user name")
```

Comment:
the help string is not very telling -- i guess it's "set password for the specified ..."